src/cmds/qstat.c

```diff
@@ -2012,6 +2100,9 @@ main(int argc, char **argv, char **envp) /* qstat */
 					errflg++;
 				}
 				break;
+			case 'E':
+				E_opt = 1;
```

Comment:
Other options in qstat have simillar behaviour

[root@centos tests]# qstat 
[root@centos tests]# qstat -s 
[root@centos tests]# qstat -f 
[root@centos tests]# qstat -a

However, the -E option is special case in qstat. One more argument(s) is/are must. 
Good catch.